<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fix-csv</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_open_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_out_filename">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fname">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>51</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_out_fname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rows_qtt">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rows:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_raw_row">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>1171</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_row">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Row:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back_row">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>320</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt; Back row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next_row">
    <property name="geometry">
     <rect>
      <x>1068</x>
      <y>320</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next row &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_filename">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="txt_filename_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>50</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="txt_current_row">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
